app/src/main/java/br/org/cesar/knot_setup_app/activity/LoginActivity.java

```diff
@@ -0,0 +1,36 @@
+package br.org.cesar.knot_setup_app.activity;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+
+import java.util.ArrayList;
+
+import br.org.cesar.knot_setup_app.fragment.login.LoginFragment;
+import br.org.cesar.knot_setup_app.model.Tab;
+import br.org.cesar.knot_setup_app.utils.Constants;
+
+public class LoginActivity extends BaseActivity {
+
+    private static final String HEADER_TITLE = "User Credentials";
+    private String gatewayName;
+    private ArrayList<Tab> tabs;
+
+    @Override
+        protected void onCreate(Bundle savedInstanceState) {
+            super.onCreate(savedInstanceState);
+
+            tabs = new ArrayList<>();
+            gatewayName = getIntent().getStringExtra(Constants.GATEWAY_ID);
+
+            Tab tab = new Tab(gatewayName, new LoginFragment());
+            tabs.add(tab);
+
+            setupHeader(HEADER_TITLE,tabs);
+    }
+
+    @Override
```

Comment:
If only calling the super method, no need to have this overriden

Reply:
The method was not necessary, removed.